index.html

```diff
@@ -918,6 +937,37 @@ <h1 class="editableArea">Headline1</h1>
 						</div>
 					</div>
 				</li>
+
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
```

Comment:
It is, but because they're narrow and there's 4 of them, it just looked a lot lot better to have 2 side by side.